packages/editor/src/components/block-list/index.js

```diff
@@ -182,37 +193,54 @@ class BlockList extends Component {
 			blockClientIds,
 			rootClientId,
 			isDraggable,
+			selectedBlockClientId,
+			multiSelectedBlockClientIds,
 		} = this.props;
 
 		return (
 			<div className="editor-block-list__layout">
-				{ map( blockClientIds, ( clientId, blockIndex ) => (
-					<BlockListBlock
-						key={ 'block-' + clientId }
-						index={ blockIndex }
-						clientId={ clientId }
-						blockRef={ this.setBlockRef }
-						onSelectionStart={ this.onSelectionStart }
-						rootClientId={ rootClientId }
-						isFirst={ blockIndex === 0 }
-						isLast={ blockIndex === blockClientIds.length - 1 }
-						isDraggable={ isDraggable }
-					/>
-				) ) }
+				{ map( blockClientIds, ( clientId, blockIndex ) => {
+					return (
+						<AsyncModeProvider
+							key={ 'block-' + clientId }
+							value={
+								selectedBlockClientId !== clientId &&
```

Comment:
What is this value representing? For clarity and legibility, it'd be deserving of being assigned a variable.

Reply:
It represents when we enable/disable the async mode, I can add a clarifying comments. The idea is that selected blocks are sync, the others are async.

Reply:
Well, from how I interpret it, it represents whether the rendered block occurs within the current selection, something akin to a `isBlockInSelection` variable (or even selector, though I don't think this can help here as written).